src/styles/blocks/main.scss

```diff
@@ -0,0 +1,10 @@
+body {
+  margin: 0;
+}
+
+html {
+  color: $secondary-color;
```

Comment:
The variable `$secondary-color` is used here. Ensure that this variable is defined in your SCSS files. If it's not defined, the SCSS compilation will fail.